assets/css/style.scss

```diff
@@ -201,6 +201,9 @@
 	.wc-block-components-product-sale-badge {
 		line-height: 1;
 	}
+	.editor-styles-wrapper .wp-block-button .wp-block-button__link:not(.has-text-color) {
```

Comment:
I wonder if the `.wp-block-button class` is really needed here. the shorter the selector the better:
```css 
.editor-styles-wrapper .wp-block-button__link:not(.has-text-color) {
}
```

Reply:
I've done it this way to match the specificity in Twenty Nineteen. This was how many selectors there were in the upstream fix I made. See here: https://github.com/WordPress/wordpress-develop/pull/1012/files#diff-132d2e4e3a183056d976fbb84a21b64fb2295266219ff0e94375f102ca35f7d9R972